src/patternfly/demos/Toolbar/examples/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Documentation from '@siteComponents/Documentation';
+import Example from '@siteComponents/Example';
+import toolbarSimpleExampleRaw from '!raw!./toolbar-simple-example.hbs';
+import toolbarComplexExampleRaw from '!raw!./toolbar-complex-example.hbs';
+import ToolbarSimpleExample from './toolbar-simple-example.hbs';
+import ToolbarComplexExample from './toolbar-complex-example.hbs';
+// import toolbarComplexExampleDoc from '../docs/toolbar-complex.md';
+import docs from '../docs/design.md';
+// import '../styles.scss';
+
+export const Docs = docs;
+
+export default () => {
+  const toolbarSimpleExample = ToolbarSimpleExample();
+  const toolbarComplexExample = ToolbarComplexExample();
+
+  return (
+    <Documentation docs={Docs}>
+      <Example heading="Toolbar Simple Example" handlebars={toolbarSimpleExampleRaw} minHeight="20em">
```

Comment:
curious why you're using `minHeight` here but not on the other? It doesn't look like they need a `minHeight`

Reply:
I believe this was initiated when I created the demo, removing it.